### ISSUE TITLE
Zwift Not Receiving Data

### ISSUE THREAD

**Issue by Middy2000** [cagnulein/qdomyos-zwift#2647]
QZ is connecting to NoblePro E10i treadmill but Zwift is not receiving a speed or cadence source.

Phone is Android - Samsung Galaxy - Zwift is on ATV

We have reset all settings, we have installed Zwift on a different device (Ipad).

QZ connects to treadmill.  

Have enabled virtual device Bluetooth.

Please find 2 logs, one from Thursday before all settings reset and a clean attempt today.  
[debug-Thu_Oct_10_07_22_23_2024[1].log](https://github.com/user-attachments/files/17340472/debug-Thu_Oct_10_07_22_23_2024.1.log)
[debug-Fri_Oct_11_10_06_36_2024.log](https://github.com/user-attachments/files/17340487/debug-Fri_Oct_11_10_06_36_2024.log)

**Comment by cagnulein** [comment]
thanks @Middy2000 I checked and everything looks ok from the QZ point of view.
I saw that QZ is already broadcasting metrics over bluetooth (the user didn't use the wahoo tread on zwift).
So a couple of suggestions:
- rename the android device to "run" in the android settings and reboot the phone
- use the wahoo virtual tread device in zwift instead of the bluetooth one. If he doesn't see it, check if the phone and the ipad are on the same network

let me know!
